features/steps/thens/attributes.py

```diff
@@ -61,43 +61,31 @@ def step_impl(context, entity, other_entity):
         misc.handle_errors(context, errors)
 
 
-@then('The type of attribute {attribute} should be {expected_entity_type}')
+@then('The type of attribute {attribute} must be {expected_entity_type}')
 def step_impl(context, attribute, expected_entity_type):
 
     def _():
         for inst in context.instances:
+            if isinstance(inst, tuple):
+                inst = inst[0]
```

Comment:
Just curious: What does this do?

Reply:
The `@given('Its attribute {attribute}')` given uses the below line in order to setup the scope for the following `@thens`
https://github.com/buildingSMART/ifc-gherkin-rules/blob/main/features/steps/givens/attributes.py#L51

It returns either an attribute or an attribute wrapped in tuple/list. In this example it returns for example:

Captured stdout:
(#518=IfcShapeRepresentation(#17,'Axis','Curve3D',(#383)),)

So the above is unpacked from the touple.